collectors/build/jenkins/src/main/java/com/capitalone/dashboard/collector/DefaultHudsonClient.java

```diff
@@ -378,6 +408,18 @@ private BuildStatus getBuildStatus(JSONObject buildJson) {
         }
 
     }
+    
+    private String getDomain(String url) throws URISyntaxException {
+        URI uri = new URI(url);
+        String domain = uri.getHost();
+        domain = domain.startsWith("www.") ? domain.substring(4) : domain;
+        return domain;
```

Comment:
Well, I just thought we'll remove the www if the URL has it, but it's not needed I suppose. I'll fix it.